PDSim/core/StateFlooded.pyx

```diff
@@ -0,0 +1,609 @@
+from __future__ import division
+
+cimport cython
+
+
+
+"""
+State_Flooded should receive   set_Fluid and set_Liquid as well as T,P from the 
+refrigerant, meaning the State class of CoolProp
+
+To add as inputs by the user:
+
+__hasLiquid__ == True/False
+set_Liquid()
+
+
+"""
+
+cdef class State_Flooded(State):
```

Comment:
In general this state class needs to be reworked a bit.  You need to implement the ``update`` function that sets the state in the AbstractState based on the given state variables and the oil mole fraction.  Furthermore, you need to implement all the property accessors that are implemented in the base class because you want PDSim to use your mixture properties instead of the pure fluid properties